### ISSUE TITLE
npm gets hung up on installing this package.

### ISSUE THREAD

**Issue by Arametheus** [zoonru/puphpeteer#13]
I'm encountering a problem during deployment with AWS Elastic Beanstalk. The deployment runs npm install, but it recently started hanging for hours on this step before eventually timing out and failing the deployment. This caused significant downtime, with our servers being down for two days, and it required about 15 hours on the phone with AWS support to troubleshoot.

After extensive debugging, I discovered that the issue occurs when the package "@zoon/puphpeteer": "github:zoonru/puphpeteer" is included in my package.json file. When I remove this package, the deployment works fine.

To further troubleshoot, I found that the hang happens specifically when installing Puppeteer. The strange part is that if I manually run npm install, it hangs the first time, but if I cancel and try again, it works without issues.

Additionally, it seems the package might be using an older version of Puppeteer.

Is there any known workaround or fix for this issue? Any insights or suggestions would be greatly appreciated.

we are running on EC2 linux 2023. 

if you need anything let me know.

**Comment by xtrime-ru** [comment]
Hi! Looks like you got same issue as here: https://github.com/zoonru/puphpeteer/issues/9

I ve updated docs and package.json multiple times in last month. Can you check if this works for you?
```
npm install https://github.com/zoonru/puphpeteer/tarball/zoon
```

**Comment by Arametheus** [comment]
so I Actually ended up installing puppeteer directly "puppeteer": "^23.1.0",   and not adding the zoon package to my package.json file thru the npm install command and that fixed it for me.   I will try this on my next test run and see if it fixes this and let you know though. This might be a few days as I'm working on another part of our application.

**Comment by xtrime-ru** [comment]
Yes, you can install packages directly. But there are two requirements: puppeteer and https://github.com/zoonru/rialto/

**Comment by Arametheus** [comment]
well so far npm install https://github.com/zoonru/puphpeteer/tarball/zoon seemed to have fixed it but let me run a few tests on it and make sure its not one of the flukes where it runs fine this time but next time fails.

**Comment by Arametheus** [comment]
it was working now I gets this.  almost crashed my systems again..

npm warn tarball tarball data for @zoon/puphpeteer@https://github.com/zoonru/puphpeteer/tarball/zoon (sha512-gk5Htsn0tpr2Zavj46HD+V/8NXORhYfSXh+V+n9vVfeZ4MAWAuze6MEhb95myN0oeK8UrrhoT8IQirMaggVfbQ==) seems to be corrupted. Trying again.
npm error code EINTEGRITY
npm error sha512-gk5Htsn0tpr2Zavj46HD+V/8NXORhYfSXh+V+n9vVfeZ4MAWAuze6MEhb95myN0oeK8UrrhoT8IQirMaggVfbQ== integrity checksum failed when using sha512: wanted sha512-gk5Htsn0tpr2Zavj46HD+V/8NXORhYfSXh+V+n9vVfeZ4MAWAuze6MEhb95myN0oeK8UrrhoT8IQirMaggVfbQ== but got sha512-bcCt5JEPZ9DTLKEw0A+rDtb27cydl/8kPuSxZ63FgPTmfGrh66dISkYI/TYjQqLr26hMV7ADEIbwJ9WGIY87OQ==. (2565514 bytes)



for now I forked yours and then set v2.2.0 as a release so I could link to it till you get it fixed.
"@zoon/puphpeteer": "https://github.com/Arametheus/puphpeteer/archive/refs/tags/v2.2.0.tar.gz",

**Comment by xtrime-ru** [comment]
Probably its because i pushed new commits to master. 

You can install exact tag like this: `npm install https://github.com/zoonru/puphpeteer/tarball/v2.3.0`

**Comment by xtrime-ru** [comment]
Fixed in https://github.com/zoonru/puphpeteer/issues/22 and https://github.com/zoonru/puphpeteer/pull/21
